### PR TITLE
cosmwasm: simplify quorum calculation

### PR DESCRIPTION
**Pull request by tbjump**
No functional changes. Just making a complicated formula more easy to read and make it match the formula in other places of the codebase.

### CODE REVIEW

**Review by kcsongor — APPROVED** [review]

**Review by jumpsiegel — APPROVED** [review]

File: cosmwasm/contracts/wormhole/src/state.rs [hunk 1]
```diff
@@ -238,7 +238,7 @@ impl GuardianSetInfo {
         if self.addresses.is_empty() {
             return 0;
         }
-        ((self.addresses.len() * 10 / 3) * 2) / 10 + 1
+        ((self.addresses.len() * 2) / 3) + 1
```
Comment (claudijd) [inline cosmwasm/contracts/wormhole/src/state.rs:241]: Do each of the chains we're modifying quorum calculation here have independant quorum unit-tests to ensure that this doesn't cause a regression?  I'm thinking of what exists for Solidity / Guardians:

- https://github.com/wormhole-foundation/wormhole/blob/dev.v2/node/pkg/processor/quorum_test.go
- https://github.com/wormhole-foundation/wormhole/blob/dev.v2/ethereum/forge-test/Messages.t.sol#L13-L29

Where possible, I'd love to see the core trust assumptions, like quorum, well unit-tested on all chains.  This is especially true when making medications to those implementations to help prevent regression scenarios.

Comment (claudijd) [reply cosmwasm/contracts/wormhole/src/state.rs:241]: That said, the math is pretty straight forward, so as a function of the review, I agree with the implementation changes, would just like to see associated test coverage.

**Review by kcsongor — APPROVED** [review]

**Review by evan-gray — CHANGES_REQUESTED** [review]
I would approve, but given the criticality of the code can you provide the unit tests that cover each of these?

**Comment by tbjump** [comment]
Unit tests are here: https://github.com/wormhole-foundation/wormhole/blob/dedcea34af2c6760882c019957d51359aef186f7/cosmwasm/contracts/wormhole/src/testing/tests.rs#L6

**Review by kcsongor — DISMISSED** [review]

**Review by jynnantonix — APPROVED** [review]

**Review by jynnantonix — APPROVED** [review]

**Review by jynnantonix — DISMISSED** [review]

**Comment by hendrikhofstadt** [comment]
Are we 100% sure this behaves the same in the Solana bpf runtime?

**Comment by evan-gray** [comment]
Discussed with @jumpsiegel and @hendrikhofstadt and we would prefer not to change these critical calculations at this time.

**Pull request closed without merging**